bitnami/kibana/README.md

```diff
@@ -264,6 +264,21 @@ initContainers:
    containerPort: 1234
 ```
 
+#### Add a sample Elasticsearch container as sidecar
+
+This chart requires an Elasticsearch instance to work. For production, you can use an already existing Elasticsearch instance or deploy the [Elasticsearch chart](https://github.com/bitnami/charts/tree/master/bitnami/elasticsearch) with the [`global.kibanaEnabled=true` parameter](https://github.com/bitnami/charts/tree/master/bitnami/elasticsearch#enable-bundled-kibana).
+
+For testing purpose, you can use a sidecar Elasticsearch container setting the following parameters during the Kibana chart installation:
```

Comment:
I would reword this as `for the purpose of testing`. I'd also add an empty line before the code block.